src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -78,78 +55,20 @@ Each version includes several metrics to analyze performance, user adoption, and
-    * To identify other problems from the app or server, the UI shows the average number of network requests made per app session (**Requests per session**) and the average server **Response time** as seen from the app’s perspective.
-    * To compare improvements or problems with **Memory** usage, the first four versions include metrics indicating better (green) or worse (red) performance than the previous version's.
+    * **Crash rates:** the percentage of all app sessions that crashed
+    * **HTTP errors:**  HTTP errors are an HTTP request that have a status code greater than 400
+    * **Network failures:** a network failure is an HTTP request that results in no response
+    * **Requests:** the average number of network requests made per app session
+    * **Response time:** the average server response time as seen from the app’s perspective
+    * **Memory usage:** memory occupied by the app on the active device
   </Collapser>
 </CollapserGroup>
 
-## Interpret version trend data (examples) [#examples]
-
-Here are some examples of how to interpret some of the data that appears.
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "200px" }}>
-        **Example**
-      </th>
-
-      <th>
-        **Comments**
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        Improvements
-      </td>
-
-      <td>
-        A new release with faster response time than previous releases may indicate that development efforts were successful.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Problems
-      </td>
-
-      <td>
-        A new release with a spike in crashes, as compared to the previous releases, may indicate that a change introduced in the library or framework is not handling the new scenario or code path properly.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        User adoption
-      </td>
-
-      <td>
-        A new release with low numbers of upgrades or new installs may indicate slow adoption of the newest version. This in turn may help justify a new or changed focus in marketing the new release.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Release maintenance
-      </td>
-
-      <td>
-        An older release with continued higher numbers of total users also may indicate slow adoption of the newest version. This in turn may result in unanticipated costs for continued support, or delays in deprecating the old version and its back-end APIs.
-      </td>
-    </tr>
-  </tbody>
-</table>
-
 ## Query version trend data [#insights]
 
 When querying with [NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql), we provide two attributes to help analyze version trend information:
 
 * To track when a session includes a new install or a new upgrade, use `install`. This attribute records `true` for new installations.
 * To track the last version of the mobile app when an upgrade is detected, use `upgradeFrom`.
```

Comment:
Would it be helpful to provide an actual NRQL query example? Your mobile SME could help think one up if you decide to include.